src/pie-chart/styles.scss

```diff
@@ -27,13 +27,30 @@
   }
 }
 
+.content--fit-height {
+  flex: 1;
+}
+
 .status-container {
   /* used in test utils */
 }
 
 .chart-container {
   display: flex;
   flex: 1;
+
+  &--fit-height {
+    height: 100%;
+  }
+}
+
+.chart-container-chart-plot {
+  display: contents;
+
+  &--fit-height {
+    position: absolute;
+    inset: 0;
```

Comment:
This is not supported in Safari 14.0 :( https://caniuse.com/mdn-css_properties_inset